lib/cloud_controller/metrics/periodic_updater.rb

```diff
@@ -120,7 +123,12 @@ def update_vitals
         num_cores: VCAP::HostSystem.new.num_cores
       }
 
-      @updaters.each { |u| u.update_vitals(vitals) }
+      @statsd_updater.update_vitals(vitals)
+
+      prom_vitals = vitals.clone
+      prom_vitals.delete(:uptime)
+      prom_vitals[:started_at] = @start_time.to_i
+      @prometheus_updater.update_vitals(prom_vitals)
```

Comment:
Prometheus best practice is to [emit the timestamp when something happened and not the time since something happened](https://prometheus.io/docs/practices/instrumentation/#timestamps-not-time-since). The uptime can then be calculated by the monitoring system.